src/routes/upcoming.jsx

```diff
@@ -15,18 +16,13 @@ export default function Upcoming() {
       });
   };
 
-const handleDelete = (id) => {
-    fetch(`http://localhost:5001/api/v1/destinations/${id}`, {
-    method: 'DELETE'
-    })
-    .catch(error => console.error(error));
-  }
-
-
   useEffect(() => {
     fetchDestinationData();
   }, []);
 
+  useEffect(() => {}, [destinations])
+
+
   const handleClick = async () => {
```

Comment:
what dis do?

Reply:
^ this went to the wrong place. Meant the handle click function

Reply:
Randomly picks a destination, changed it to handleRandomDest